Description: Focal Qt is old, it doesn't have QColorConstants.
Drop this when building with Qt >= 5.14.
Author: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Forwarded: not-yet

--- a/ui/qt/qt_shim.cc
+++ b/ui/qt/qt_shim.cc
@@ -50,7 +50,7 @@
 SkColor GradientColor(const QGradient& gradient) {
   QGradientStops stops = gradient.stops();
   if (stops.empty())
-    return QColorConstants::Transparent.rgba();
+    return qRgba(0, 0, 0, 0);
 
   float a = 0;
   float r = 0;
@@ -84,7 +84,7 @@
 SkColor TextureColor(QImage image) {
   size_t size = image.width() * image.height();
   if (!size)
-    return QColorConstants::Transparent.rgba();
+    return qRgba(0, 0, 0, 0);
 
   if (image.format() != QImage::Format_ARGB32_Premultiplied)
     image = image.convertToFormat(QImage::Format_ARGB32_Premultiplied);
